code/modules/industrial_lift/tram_lift_master.dm

```diff
@@ -104,6 +104,8 @@
 	from_where = to_where
 	set_travelling(TRUE)
 	set_controls(LIFT_PLATFORM_LOCKED)
+	update_tram_doors(CLOSE_DOORS)
+	sleep(20)
```

Comment:
don't have this be a sleep or lints will keep failing

Reply:
Done